### ISSUE TITLE
Tasks export works terrible

### ISSUE THREAD

**Issue by Bogdan-D** [getting-things-gnome/gtg#159]
Version: 0.3.1
OS: Ubuntu 15.04

Here is my test task
`
Testing task
@sometag
- Subtask 1
- Subtask 2
`

If I try to export to text or html formats the result will contain some wrong information. 
Example for text:
`
TODO LIST:
* Testing task(@sometag )
    → d2d38b27-bf92-47e5-9c50-2fd8d3917147 → c636f8d1-b433-4a7f-b702-7638538e
    1008
    - Subtask 1(@sometag )

    - Subtask 2(@sometag )
`

Example for html:
![2015-07-29 05 28 52](https://cloud.githubusercontent.com/assets/7849405/8948723/c894342c-35b2-11e5-8dbd-9b0bd891663e.png)

Also pdf generation produces very bad result: 
![2015-07-29 05 33 02](https://cloud.githubusercontent.com/assets/7849405/8948775/4e7d8e62-35b3-11e5-9da2-2cbb66ad6834.png)

It's very small and at right side of paper.

**Comment by izidormatusov** [comment]
Hey Bogdan,

I suspect that the primary point of this issue is that garbage `→ d2d38b27-bf92-47e5-9c50-2fd8d3917147 → c636f8d1-b433-4a7f-b702-7638538e` isn't it?

PDF export thing is some very special format where you could print it, fold it and keep it in your pocket for quick reference. What export to PDF would you expect there?

**Comment by Bogdan-D** [comment]
Yeah, it's because of garbage. Not only like that, there also encoding problems. Here an example of Testing task with content from wikipedia main page (on russian):
![2015-07-30 23 38 40](https://cloud.githubusercontent.com/assets/7849405/8994493/595e42ec-3714-11e5-87fb-e9541b5dce98.png)

And here is result from exporting to plain text (html works correct with encoding):
![2015-07-30 23 38 23](https://cloud.githubusercontent.com/assets/7849405/8994504/67d36fd2-3714-11e5-95dd-e5b59973b8ba.png)

**Comment by izidormatusov** [comment]
Can you provide plain text so I can reproduce it myself?

BTW: Do you achieve that visual noise (->) for each task or only some selected tasks?

**Comment by Bogdan-D** [comment]
Visual noise seems like present for any task with subtasks.

Here is plain text:
```
Testing task
@sometag
- Subtask 1
- Subtask 2

Лоуренс Салливан Росс (англ. Lawrence Sullivan Ross; 27 сентября 1838, Бентонспорт, Территория Айова — 3 января 1898, округ Бразос, Техас) — американский политик, генерал армии КША во время Гражданской войны, 19-й губернатор Техаса, президент Сельскохозяйственного и механического колледжа Техаса.

Росс вырос в Республике Техас, большая часть его детства прошла на границе с Мексикой, где его предки были одними из основателей города Уэйко. После школы он поступил в Бэйлорский университет и во время летних каникул принял участие в сражении с команчами, в котором был тяжело ранен. Окончив образование, Росс вступил в ряды техасских рейнджеров и в 1860 году отличился в битве у реки Пиз, где он «спас» Синтию Паркер, захваченную команчами ещё ребёнком.

Когда Техас стал частью КША, Росс присоединился к армии конфедератов. Он принял участие в 135 сражениях и стал одним из самых молодых генералов конфедерации. После Гражданской войны Росс непродолжительное время служил шерифом округа Мак-Леннан и, выйдя в отставку в 1875 году, был избран делегатом конституционного конвента Техаса. За исключением двух лет, когда он служил сенатором штата, Росс следующее десятилетие провёл на своём ранчо. В 1887 году он был избран 19-м губернатором Техаса. За два срока, проведённых на этом посту, Росс инициировал постройку нового Капитолия Техаса, прекратил «войну соек и дятлов (англ.)русск.».

Несмотря на популярность, Росс отказался баллотироваться на третий губернаторский срок. Через несколько дней после ухода с должности он стал президентом Сельскохозяйственного и механического колледжа Техаса (ныне Техасский университет A&M). Россу приписывают спасение колледжа от закрытия, при нём было построено много новых зданий и родились многие университетские традиции. После смерти Росса легислатура Техаса инициировала создание Педагогического колледжа, названного его именем.
```

**Comment by nekohayo** [comment]
This issue here is up for grabs for anyone wanting to see it happen. Please contribute code (or bring in someone who can) to the project for this purpose. Otherwise, I doubt this will move forward.

**Comment by pentux-GitHub** [comment]
Housekeeping note: the export plugin is being redesigned for 0.8 (#299, together with the template issues #828–#832). It seems this would best be handled as part of that effort rather than patched in the current plugin — linking it here so it's on the radar when that work starts.
